src/components/molecules/LiveSchedule/LiveEvent.tsx

```diff
@@ -53,7 +53,7 @@ export const LiveEvent: FC<LiveEventProps> = ({ venue, event }) => {
         </div>
         <div className="schedule-event-info-room">
           <div onClick={enterLiveEvent}>
-            {event.room ?? "Enter"} - {venue.name}
+            {event.room ?? "Enter"} {venue && `- ${venue.name}`}
           </div>
```

Comment:
[nit] We should extract this out into a common component as we're using the same code in `src/components/molecules/EventDisplay/EventDisplay.tsx`

There might even be a more encompassing' chunk we can extract too, I didn't look too closely.

Reply:
Yep definitely opportunities to zoom out on DRYing up. We should DRY at a business-component level (eg. One unified video chat room component that’s reusable everywhere), that’ll guide decisions where to DRY vs rebuild.

Reply:
_nods_ that definitely makes sense to me.